testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListViewModel.kt

```diff
@@ -17,7 +15,7 @@ class MiniAppListViewModel constructor(
     constructor() : this(MiniApp.instance())
 
     private val _miniAppListData =
-        MutableLiveData<List<MiniAppInfo>>().apply { value = emptyList() }
+        MutableLiveData<List<MiniAppInfo>>()
```

Comment:
I think it is better to keep it initialized with an emptyList() because the object has been created already. When it is an empty list, the adapter wouldn't do much for the list.